cmd/sonobuoy/app/e2e.go

```diff
@@ -75,12 +75,10 @@ func e2es(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 	defer gzr.Close()
-	restConfig, err := e2eflags.kubecfg.Get()
```

Comment:
This might be a slightly deeper change than I originally expected. Since e2e does many things (oops) it will need the config later on. Would you be able to move this down below `if !e2eflags.rerun {` this line?

There is a flag, which needs rethinking, `--rerun` which allows users to rerun the failed tests if an error is detected.

Reply:
done, rather than moving it down I have added a check.

Moving down was causing the initiation of client twice, so added a check rather.